include/libopencm3/cm3/common.h

```diff
@@ -111,4 +111,26 @@
 #define BIT30 (1<<30)
 #define BIT31 (1<<31)
 
+/* Generic bit mask manipulation */
+/** Set a \p field inside of \p reg_ with bits_. Assumes that the field name can
```

Comment:
In general, there's no poitn in having a "non-doxygen" and then also a doxygen comment.  Just one doxygen comment.